<!doctype html>
<html lang="en">

<head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
    <link rel="stylesheet" href="/styles/styles.css">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="/styles/fa.css">

    <title>Marc Expòsit Goy</title>

</head>

<body>

    <div class="title-pad d-none d-md-block">
        <div class="container text-center">
            <a class="WS_title" href="/index.html">
                Marc Expòsit Goy</a>
        </div>
    </div>


    <nav class="navbar navbar-expand-md navbar-light bg-opac">

        <a class="navbar-brand d-md-none WS_title" href="/index.html">Marc Expòsit Goy</a>

        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav ml-auto mr-auto">
                <li class="nav-item pr-5 pl-5">
                    <a class="nav-link" href="/index.html">Home</a>
                </li>
                <li class="nav-item pr-5 pl-5 active">
                    <a class="nav-link" href="#">Research<span class="sr-only">(current)</span></a>
                </li>
                <!--li class="nav-item pr-5 pl-5">
                    <a class="nav-link" href="/outreach.html">Outreach</a>
                </li-->
                <li class="nav-item pr-5 pl-5">
                    <a class="nav-link" href="/resume.html">Resume</a>
                </li>
                <li class="nav-item pr-5 pl-5">
                    <a class="nav-link" href="/contact.html">Contact</a>
                </li>
            </ul>
        </div>
    </nav>

    <div class="container content">

        <!--div class="row" id="ResearchIntro">
            <div class="col-12 py-3">
                <p>
                    I am interested in answering how synbio can advance medicine.
                </p>
            </div>
        <div-->
        <div class="row" id="TransSynBio">
            <div class="col-12 py-3">
                <h5>Translational Synthetic Biology lab</h5>
                <p><i>Sept 2019 - Present, Pompeu Fabra University (Spain)</i></p>
                <p>To strengthen my knowledge on synthetic biology and medicine, and complement it with computational
                    approaches, I am actually studying for the MSc in Bioinformatics and Biostatistics of the University
                    of Barcelona. For my master's thesis,
                    I am working full time in the laboratory of <a href="https://www.upf.edu/web/synbio">Prof. Marc
                        Güell</a>, who was a postdoc at Harvard's
                    genetics professor George Church's laboratory before starting his own lab in the Pompeu Fabra
                    University. There, I am applying recent
                    developments in the field of <b>synthetic biology to solve health problems</b> such as muscular
                    distrophy or the need for new antibiotics. The projects I am involved with are:</p>
                <ul>
                    <li>
                        <i>Using machine learning to predict the efficiency of CRISPR-Cas9 <b>genome engineering</b> in
                            vivo</i>. By studying the mutations introduced by CRISPR-Cas9, prime editing or base editors
                        at multiple sites of the genome, we are
                        building a machine learning model to estimate the outcomes of genome engineering with each
                        method. Hence, I am learning to use the most advanced genome engineering tools, to create
                        adeno-associated viruses (AAVs) for their delivery,
                        and to analyze sequencing data and interpret it to create a computational model.
                    </li>
                    <li>
                        <i>Cell-free biosynthetic pathway engineering for the identification on <b>novel
                                antibiotics</b></i>. We have developed a cell-free system with the metabolic pathways
                        needed for the synthesis of an antimicrobial compound. We are
                        using Multiplex Automated Genome Engineering (MAGE) to generate variants in the enzymes involved
                        in this metabolic route with the objective of producing novel compounds with unique
                        antimicrobial activities.
                    </li>
                    <li>
                        <i>Increasing lentiviral payload capacity to use them as gene therapy for <b>muscular
                                dystrophy</b></i>. Introducing variants by MAGE in lentiviral vectors we are increasing
                        the size of the cassettes this vectors can carry. This
                        will allow the delivery of larger transgenes, such as LAMA2 which could cure myotonic dystrophy
                        type 2.
                    </li>
                </ul>
                <p class="mb-0"><i>Related publications:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>Hodgson, J. CRISPR target prediction remains blunt tool for clinical applications. <i>Nat
                            Biotechnol</i> 37, 204–205 (2019) doi:<a
                            href="https://www.nature.com/articles/s41587-019-0057-7">10.1038/s41587-019-0057-7</a>
                    </li>
                    <li>Peng, H.; Zheng, Yi; <i>et al.</i> Recognition of CRISPR/Cas9 off-target sites through ensemble
                        learning of uneven mismatch distributions. <i>Bioinformatics</i> 34, 757-765 (2018) doi:<a
                            href="https://doi.org/10.1093/bioinformatics/bty558">10.1093/bioinformatics/bty558</a>
                    </li>
                    <li>Fang, Lei; Guell, Marc; Church, George M.; Pfeifer, Blaine A. Heterologous erythromycin
                        production across strain and plasmid construction. <i>Biotech Prog</i> 34(1), 271-276 (2017)
                        doi:<a
                            href="https://aiche.onlinelibrary.wiley.com/doi/abs/10.1002/btpr.2567">10.1002/btpr.2567</a>
                    </li>
                    <li>Counsell, J. R. <i>et al.</i> Lentiviral vectors can be used for full-length dystrophin gene
                        therapy.
                        <i>Sci Rep</i> 7, 44775 (2017); doi:<a
                            href="https://www.nature.com/articles/srep44775">10.1038/srep44775</a></li>
                </ol>
            </div>
        </div>
        <!--/row-->

        <div class="row" id="GehrkeLab">
            <div class="col-12 py-3">
                <h5>Gehrke lab, MIT</h5>
                <p><i>Sept. 2018 - July 2019, Massachusetts Institute of Technology (MA, USA)</i></p>
                <p>For my thesis of the MSc in Bioengineering, I worked under the supervision of <a href="https://www.gehrkelab.org/">Prof. Lee
                        Gehrke</a>, reducing
                    the development time and cost of <b>paper-based
                        diagnostics for emerging viruses</b> like Dengue or Zika. Current production methods rely on the
                    hybridoma technique for the identification of monoclonal antibodies binding to viral proteins. This
                    can take up to 8 months and requires animal immunization, limiting the applicability of the tests
                    for emerging viruses that could spread at any time. My knowledge in genetic engineering allowed us
                    to use a yeast surface display library (created by Prof. Kruse and colleagues at Harvard Medical
                    School) to identify nanobodies and
                    develop tests for new pathogens in about one month. In addition, I improved the current yeast
                    selection method by adding a quantitative step and characterized every step of the process so that
                    it could be implemented elsewhere. I also designed and implemented a nanobody cloning method to
                    produce them as recombinant proteins in <i>E. coli</i> or HEK293F cells. To culminate the project, I
                    presented a poster in the Harvard Microbiology Department Retreat, where I had the chance to discuss
                    the results with faculty and help other people implement the nanobody selection process in their
                    labs. The title of the master thesis is: “Rapid identification of Nanobody Reagents for Emerging
                    Viruses using Yeast Surface Display”. It was awarded with the Salvador Gil Prize to the best thesis
                    of the MSc in Bioengineering of the Ramon Llull University.</p>
                <p class="mb-0"><i>Poster presentations:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>
                        <b>M. Exposit Goy</b>, A. Mondini, M. Roberts, A. Roffler, <u>A. Schmidt, L. Gehrke</u>.
                        Selection and characterization of Nanobody Reagents for Emerging Viruses, Poster presented at
                        the Harvard Microbiology Deparment Retreat (July 2019)
                    </li>
                </ol>
                <p class="mb-0"><i>Related publications:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>Bosch, I. <i>et al.</i> Rapid antigen tests for dengue virus serotypes and Zika virus in patient
                        serum.
                        <i>Sci Transl Med</i> 9, eaan1589 (2017) doi:<a
                            href="https://stm.sciencemag.org/content/9/409/eaan1589">10.1126/scitranslmed.aan1589</a>
                    </li>
                    <li>McMahon, C. <i>et al.</i> Yeast surface display platform for rapid discovery of conformationally
                        selective nanobodies.
                        <i>Nat Struct Mol Biol</i> 25, 289–296 (2018) doi:<a
                            href="https://www.nature.com/articles/s41594-018-0028-6">10.1038/s41594-018-0028-6</a>
                    </li>
                </ol>
            </div>
        </div>
        <!--/row-->

        <div class="row" id="BioIQSIGEMTeam">
            <div class="col-12 py-3">
                <h5>BioIQS iGEM team</h5>
                <p><i>Feb. 2018 - Sept. 2018, IQS School of Engineering (Ramon Llull University, Spain)</i></p>
                <p>I created a team to compete in the 2018 <a href="https://igem.org/Main_Page">iGEM competition</a>
                    with the objective of experiencing how to
                    have <b>an impact in society through synthetic biology</b>. From February 2018 to October 2018, I
                    led a
                    team of 12 master’s students to develop a personalized gluten sensor for celiac patients. Our sensor
                    was based on the patient-specific variant of the HLA-DQ protein, which recognizes gluten epitopes
                    and determines the sensibility of each patient. To build it, we isolated by PCR the <i>hla-dq</i>
                    gene from
                    saliva samples and expressed the regions involved in gluten detection. I worked on the design of the
                    genetic system to produce recombinant HLA-DQ, which was challenging because the receptor consists of
                    two polypeptide chains. The simple method we created not only facilitates the construction of the
                    sensor but could also be used to study the interaction between HLA-DQ and gluten peptides at the
                    structural level. This project taught me soft skills such as leadership, communication and team
                    management. We collected 7,150.00€ ($7,950.00) from investors to develop our project and presented
                    it at the
                    <a href="http://2018.igem.org/Main_Page">2018 Giant Jamboree</a>, in front of more than 300 iGEM
                    teams from all around the world. Our project was
                    awarded the bronze medal of the competition.</p>

                <p class="mb-0"><i>Oral and poster presentation:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>
                        <b>Exposit Goy, M.</b>; Carol, X.; Chi, J.; Clariana, M.; Compte, R.; Garcia, C.; Jorda, M.;
                        Martinez, E.; Maull, V.; Quintanas, J.; Sola, M.; Vega, A.; <u>BioIQS iGEM
                            team</u>. Personalized diagnostic kit for celiac disease. Oral presentation and poster
                        session at
                        the 2018 iGEM Giant Jamboree (October 2018)
                    </li>
                </ol>
                <p class="mb-0"><i>Project website:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>BioIQS 2018 iGEM team: A personalized gluten sensor for celiac disease patients. <a
                            href="http://2018.igem.org/Team:BioIQS-Barcelona">Link to website</a>
                    </li>
                </ol>
            </div>
        </div>
        <!--/row-->


        <div class="row" id="Lequia">
            <div class="col-12 py-3">
                <h5>Chemical and Environmental Engineering laboratory, LEQUIA</h5>
                <p><i>June 2016 - Sept. 2017, University of Girona (Spain)</i></p>
                <p>For my senior, undergraduate thesis, I characterized the performance of a natural microbial isolate
                    in a bioelectrochemical reactor. The reactor was fed with waste CO2 and electricity from renewable
                    energy sources and the
                    microorganism used them to produce ethanol. Since, initially, ethanol production was low, I
                    carefully studied all the factors that could be influencing its production, read literature at
                    the intersection of bioreactors engineering, metabolism, and thermodynamics, and autonomously
                    interpreted experimental data to propose technical improvements. Specifically, my contributions
                    changed the focus of the project from the identification of the ideal bioreactor conditions to the
                    optimization of metabolic pathways. With further research, this approach led the group to increase
                    ethanol production. The title of the bachelor thesis I
                    completed at <a href="http://www.lequia.udg.edu/">LEQUIA</a> is “Ethanol production from carbon
                    dioxide by an enriched culture of the isolate
                    I19 using microbial electrochemical technologies”. The thesis was qualified with excellent and
                    received a prize to the best undergraduate thesis on carbon capture technologies in Spain by the
                    PTECO2 foundation.</p>
                <p class="mb-0"><i>Related publications:</i></p>
                <!--ul style="list-style-type:none;"-->
                <ol class="papersRef">
                    <li>Blasco-Gómez, R.; <i>et al.</i> Unravelling the factors that influence the bio-electrorecycling
                        of carbon dioxide towards biofuels. <i>Green Chem</i> 21, 684-691 (2019) doi:<a
                            href="https://pubs.rsc.org/en/content/articlelanding/2019/gc/c8gc03417f">10.1039/C8GC03417F</a>
                    </li>
                    <li>Ramió-Pujol, Sara; Ganigué, Ramon; Bañeras, Lluís; Colprim, Jesús. Effect of ethanol and butanol
                        on autotrophic growth of model homoacetogens. <i>FEMS Microbiol Lett</i> 365(10), fny084 (2018)
                        doi:<a
                            href="https://academic.oup.com/femsle/article/365/10/fny084/4956523">10.1093/femsle/fny084</a>
                    </li>
                    <li>Vassilev, I.; <i>et al.</i> Microbial electrosynthesis of isobutyric, butyric, caproic acids,
                        and corresponding alcohols from carbon dioxide. <i>ACS Sustainable Chem Eng</i> 6(7), 8485-8493
                        (2018) doi:<a
                            href="https://pubs.acs.org/doi/10.1021/acssuschemeng.8b00739">10.1021/acssuschemeng.8b00739</a>
                    </li>
                </ol>
            </div>
        </div>
        <!--/row-->
    </div>
    <!--container-->
    <div class="footer">
        <div class="container">
            <div class="row text-center">
                <div class="col-12 py-5">
                    <a href="https://twitter.com/marc_exposit"><i class="fab fa-twitter"></i></a>
                    <a href="https://www.linkedin.com/in/mexposit/"><i class="fab fa-linkedin"></i></a>
                </div>
            </div>
        </div>
    </div>



    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
        crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
        crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
        crossorigin="anonymous"></script>
    <script src="function.js"></script>
</body>

</html>